### PR TITLE
Add appeal to kindness to dispenser to help prevent abuse.

### PR DESCRIPTION
**Pull request by egieseke**
## Summary

Add Appeal to kindness to dispenser HTML.
Expand the size of the address text field to 80 characters.

## Test Plan

Tested in the browser.

### CODE REVIEW

File: cmd/dispenser/server.go [hunk 1]
```diff
@@ -104,7 +104,12 @@ const topPageTemplate = `
     <div class="g-recaptcha" data-sitekey="{{.RecaptchaSiteKey}}">
     </div>
     <div>
-      <input id="target" placeholder="target address">
+      The dispensed Algos have no monetary value and should only be used to test applications.
+      <p>This service is gracefully provided to enable development on the Algorand blockchain test networks.
+      <p>Please do not abuse it by requesting millions of Algos.
```
Comment (onetechnical) [inline cmd/dispenser/server.go]:
```suggestion
      <p>Please do not abuse it by requesting more Algos than you need for testing.
```

File: cmd/dispenser/server.go [hunk 2]
```diff
@@ -104,7 +104,12 @@ const topPageTemplate = `
     <div class="g-recaptcha" data-sitekey="{{.RecaptchaSiteKey}}">
     </div>
     <div>
-      <input id="target" placeholder="target address">
+      The dispensed Algos have no monetary value and should only be used to test applications.
+      <p>This service is gracefully provided to enable development on the Algorand blockchain test networks.
```
Comment (onetechnical) [inline cmd/dispenser/server.go]:
```suggestion
      <p>This service is gracefully provided to enable development on the Algorand blockchain test networks.</p>
```

File: cmd/dispenser/server.go [hunk 3]
```diff
@@ -104,7 +104,12 @@ const topPageTemplate = `
     <div class="g-recaptcha" data-sitekey="{{.RecaptchaSiteKey}}">
     </div>
     <div>
-      <input id="target" placeholder="target address">
+      <p>The dispensed Algos have no monetary value and should only be used to test applications.</p>
+      <p>This service is gracefully provided to enable development on the Algorand blockchain test networks.</p>
+      <p>Please do not abuse it by requesting millions of Algos.</p>
```
Comment (onetechnical) [inline cmd/dispenser/server.go:109]:
```suggestion
      <p>Please do not abuse it by requesting more Algos than needed for testing.</p>
```

**Review by onetechnical — APPROVED** [review]
Minor suggestion

**Review by onetechnical — APPROVED** [review]

**Review by algobarb — APPROVED** [review]
Looks good to me.

**Pull request merged**